### ISSUE TITLE
Support for ios-arm64_x86_64-simulator

### ISSUE THREAD

**Issue by ToineHulshof** [CubeStuffs/tnoodle-lib-native#7]
It would be nice if the compiled xcframework includes a binary for ios-arm64_x86_64-simulator to allow for running in a simulator on an Apple silicon equipped device. Currently, in this case, the following linker error is emitted:

`... found architecture 'x86_64', required architecture 'arm64'`

**Comment by xbjfk** [comment]
Yep, this is a known issue - in the coming few weeks I may have an alternative method and arm64 simulator support will be one of the upsides - the current approach is full of hacks and magic
